Source/WebKit/UIProcess/ios/WKContentViewInteraction.mm

```diff
@@ -4690,7 +4694,7 @@ - (void)changeSelectionWithGestureAt:(CGPoint)point withGesture:(UIWKGestureType
 
 - (void)changeSelectionWithGestureAt:(CGPoint)point withGesture:(UIWKGestureType)gestureType withState:(UIGestureRecognizerState)state withFlags:(UIWKSelectionFlags)flags
 {
-    logTextInteractionAssistantSelectionChange(__PRETTY_FUNCTION__, _textInteractionAssistant.get(), point, state, gestureType, std::nullopt, flags);
+    logTextInteractionAssistantSelectionChange(__PRETTY_FUNCTION__, _textInteractionLoupeGestureRecognizer.get(), point, state, gestureType, std::nullopt, flags);
```

Comment:
I would've said remove `__PRETTY_FUNCTION__` and just use `std::source_location` but alas... rdar://116228776